### PR TITLE
[WIP]Add_pytorch_model_to_inference_model_in_scala

### PR DESCRIPTION
**Pull request by ManfeiBai**
Add PyTorch model to inference model in scala without test

### CODE REVIEW

**Comment by qiyuangong** [comment]
Too many file changes, pls rebase with master, rather than merge with master.

**Comment by ManfeiBai** [comment]
> Too many file changes, pls rebase with master, rather than merge with master.

Thanks for your comments~ Rebase soon~  :)

**Comment by ManfeiBai** [comment]
> > Too many file changes, pls rebase with master, rather than merge with master.
> 
> Thanks for your comments~ Rebase soon~ :)

Done

**Comment by qiyuangong** [comment]
> > > Too many file changes, pls rebase with master, rather than merge with master.
> > 
> > 
> > Thanks for your comments~ Rebase soon~ :)
> 
> Done

Great! Add @qiuxin2012 for review

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala [hunk 1]
```diff
@@ -232,4 +234,60 @@ class TorchModelSpec extends ZooSpecHelper{
...
+      val code = lenet +
+        s"""
+           |model = LeNet()
+           |torch.save(model, "$tmpname", pickle_module=zoo_pickle_module)
+           |""".stripMargin
+      PythonInterpreter.exec(code)
+      val model = new InferenceModel()
+      model.doLoadPyTorch(tmpname)
+
+      val genInputCode =
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala:250]: I think you should pass the input in scala.
```
val input = Tensor[Float](4, 1, 28, 28).rand()
model.doPredict(input)
```

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala [hunk 2]
```diff
@@ -232,4 +234,60 @@ class TorchModelSpec extends ZooSpecHelper{
...
+             |torch.save(model, bys, pickle_module=zoo_pickle_module)
+             |bym = bys.getvalue()
+             |""".stripMargin
+        PythonInterpreter.exec(code)
+
+        val bys = PythonInterpreter.getValue[Array[Byte]]("bym")
+        val model = new InferenceModel()
+        model.doLoadPyTorchBytes(bys)
+
+        val genInputCode =
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala:281]: same

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala (hunk 1 above)
Comment (ManfeiBai) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala:250]: Thanks! qiuxin, I would try it soon~

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala (hunk 2 above)
Comment (ManfeiBai) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/net/TorchModelSpec.scala:281]: Thank! qiuxin, I would try it soon~

**Pull request closed without merging**